rllib/trainer/pg/default_config.py

```diff
@@ -0,0 +1,16 @@
+from ray.rllib.agents.trainer import with_common_config
```

Comment:
Would like to start separating the default config for each algo from the rest of the files.